### ISSUE TITLE
Exclude logback configuration

### ISSUE THREAD

**Issue by nmiljkovic** [CurrencyCloud/currencycloud-java#103]
Hello CurrencyCloud team :),

You should consider removing `logback.xml` from the package and instead letting users configure their own logging.

1. Library users might not use logback,
2. Library users might want to have their own logback config (eg. structured JSON logging).

I encountered an issue when setting up a Spring Boot app which didn't appear to log anything, traced it to currencycloud-java. :)

**Comment by gyorgy-marbella** [comment]
I have the same problem. @nmiljkovic how did you solve this problem?

**Comment by nmiljkovic** [comment]
@gyorgy-marbella When loading configuration, any classpath resources which are in your project will have priority. I have simply added `src/main/resources/logback.xml` with my own logback configuration:

```xml
<configuration>
    <include resource="org/springframework/boot/logging/logback/base.xml"/>

    <!-- configure anything special here ->
</configuration>
```

**Comment by gyorgy-marbella** [comment]
thanks for the quick reply @nmiljkovic . We are using the spring logback extension (logback-spring.xml) and it is ignored if I add logback.xml

**Comment by nmiljkovic** [comment]
@gyorgy-marbella yes, unfortunately `logback-spring.xml` does not load because the priority is always taken by `logback.xml` from `currencycloud-java`. :(

I have not been able to find a workaround regarding this and make `logback-spring.xml` work.

**Comment by gyorgy-marbella** [comment]
I found a workaround, added the following to the application.yaml

```
logging:
  config: classpath:logback-spring.xml
```
